README.md

```diff
@@ -1,2 +1,5 @@
 # fpl-napari
-Adding fastplotlib as a front-end option for napari
+Prototype for adding `fastplotlib` as a front-end viewer for `napari`. Would allow `napari` to be used in the browser :)
```

Comment:
```suggestion
Prototype for using [`fastplotlib`](https://github.com/fastplotlib/fastplotlib) as a graphics/plotting backend for `napari`. Would allow `napari` to be used in the browser :)
```